docs_app/content/navigation.json

```diff
@@ -54,7 +54,12 @@
 			"url": "code-of-conduct",
 			"title": "Code of Conduct",
 			"tooltip": "Code of Conduct"
-		}
+    },
```

Comment:
And could you switch this with the code of conduct. I would like to have it at the very bottom